config/config.json

```diff
@@ -145,7 +145,9 @@
         "UsernameAttribute": "",
         "IdAttribute": "",
         "SkipCertificateVerification": false,
-        "QueryTimeout": 60
+        "QueryTimeout": 60,
+        "LoginFieldName": "",
```

Comment:
Nope, it's LDAP only so I'm pretty sure it should go here. Also there isn't really a Branding section, the branding stuff is just under Team Settings.